bazel/external/BUILD

```diff
@@ -0,0 +1,15 @@
+# Use a wrapper cc_library with an empty source source file to force
+# compilation of other cc_library targets that only list *.a sources.
+cc_library(
+    name = "all_external",
+    srcs = [":empty.cc"],
```

Comment:
Do you even need `empty.cc`? I was under the impression that `cc_library` targets could be used to aggregate libraries without providing their own sources.

Reply:
See the comment. An empty .cc file is needed to make Bazel trigger a build. A cc_library that depends only on .a files will report "nothing to do" if you try to build it on the command line.